### PR TITLE
Feature/quality requirements doc

### PR DESCRIPTION
**Pull request by ShamaKamina**
- Please review the architectural doc
- Still needed to be included is the diagram and technology choices

### CODE REVIEW

**Review by hlokomani — APPROVED** [review]
There are some things that can be changed with the diagrams, but I will update those or whoever is free can update them at a later stage. More specifically the ones that touch on backend. Since we are Agile, we can come back to it. Hopefully by the end of the day.

File: documentation/images/diagrams/Multi Tier.jpg
Comment (hlokomani) [inline documentation/images/diagrams/Multi Tier.jpg:1]: So we'll probably need to update this

File: documentation/images/diagrams/Pipe&Filter.jpg
Comment (hlokomani) [inline documentation/images/diagrams/Pipe&Filter.jpg:1]: Not sure about this one. All I know is that we didn't code a pipe and filter

File: documentation/images/diagrams/Multi Tier.jpg
Comment (ShamaKamina) [reply documentation/images/diagrams/Multi Tier.jpg:1]: yeah please update it

Comment (hlokomani) [reply documentation/images/diagrams/Multi Tier.jpg:1]: i'll get around to it later

**Pull request merged**